openff/bespokefit/executor/services/_settings.py

```diff
@@ -65,6 +65,9 @@ class Settings(BaseSettings):
     BEFLOW_OPTIMIZER_WORKER_MAX_MEM: Union[float, Literal["auto"]] = "auto"
     BEFLOW_OPTIMIZER_KEEP_FILES: bool = False
```

Comment:
@Yoshanuikabundi a similar option is already exposed here which causes the forcebalance files to be kept rather than cleaned up, happy to change to a more general name that includes all temp files but we probably don't need both?

Reply:
I agree but this would be a breaking change, and we use that environment variable a lot in demonstrations and troubleshooting. Maybe we should deprecate the old name and temporarily make the old name an alias of the new behaviour until we remove it?